README.md

```diff
@@ -1,87 +1,69 @@
 
-## Running
+## Executando
 
-To run the API locally in your machine, you can run the following command:
+Para executar a API localmente em sua máquina, use o seguinte comando:
 
 ```
 make run
 ```
 
-This command will start all containers necessary to run the API. In other words,
-it starts the database and the API container. If everything goes fine, you 
-should be able to query the API at `localhost:8080/gazettes/<City IBGE Code>`
-
-NOTE: When you want to restart the API, just quit the API process and 
-execute `make rerun` again. You do not need to restart the database.
+Esse comando vai iniciar todos os contêineres necessários para executar a API. Ou seja, ele inicializa o banco de dados e o contêiner da API. Se tudo correr bem, você poderá fazer consultas à API em `localhost:8080/gazettes/<City IBGE Code>`
 
-You can all check the interactive documentation at `localhost:8080/docs`. Using 
-the docs page, you can all send request to the API. But to see it working you
-need to insert data into the database. There is another make target, `make apisql`,
-which open the `psql` and connect to the database. Thus, you can insert data
-using some `INSERT INTO ...` statements and test the API. ;)
+ATENÇÃO: Quando você precisar reiniciar a API, apenas interrompa o processo da API e execute o `make rerun` novamente. Não é necessário reiniciar o banco de dados.
+Você pode checar toda a documentação interativa da API em  `localhost:8080/docs`. Nessa página, você pode fazer requisições à API diretamente. Mas, para vê-la funcionar, você precisa inserir dados no banco de dados. Há outro comando, `make apisql`, que abre o `psql` e conecta ao banco de dados. Assim, você pode inserir dados usando alguns `INSERT INTO ...` para testar a API  ;)
 
+### Usando o endpoint de ‘sugestões’
 
-### Using suggestion endpoint
+O endpoint de sugestões no Querido Diário é uma forma de coletar feedback dos usuários e usa o serviço do Mailjet para enviar e-mails. É necessário criar um token de acesso em [Mailjet](www.mailjet.com) para executar a aplicação e enviar e-mails  (salve em `config/current.env`).
 
-You need to create a token at [Mailjet](www.mailjet.com) to run
-application and send email (put on `config/current.env`).
+## Testes
 
-## Tests
+Este projeto foi desenvolvido com o paradigma TDD (Test Driven-Development). Isso significa que não há mudanças sem testes. Devemos sempre buscar ter uma cobertura de 100% do código-fonte. Outra maneira de encarar os testes é o seguinte:
 
-The project uses TDD during development. This means that there are no changes 
-without tests. We should always seek 100% source code coverage. Another way to
-think about tests is the following:
+>  “Escreva o teste que o força a escrever o código que você já sabe que quer escrever” 
 
->  "Write the test which forces you to write the code which you already know that you wanna write."
-> By Robert C. Martin (a.k.a. Uncle Bob)
+> Por Robert C. Martin (a.k.a. Uncle Bob), tradução nossa.
```

Comment:
Sem a linha em branco no meio acho que fica melhor, o que acha?

> “Escreva o teste que o força a escrever o código que você já sabe que quer escrever”

> Por Robert C. Martin (a.k.a. Uncle Bob), tradução nossa.

ou 

> “Escreva o teste que o força a escrever o código que você já sabe que quer escrever”
> Por Robert C. Martin (a.k.a. Uncle Bob), tradução nossa.

---
### English

I think it's better without the blank line, what do you think?

> “Escreva o teste que o força a escrever o código que você já sabe que quer escrever”

> Por Robert C. Martin (a.k.a. Uncle Bob), tradução nossa.

or

> “Escreva o teste que o força a escrever o código que você já sabe que quer escrever”
> Por Robert C. Martin (a.k.a. Uncle Bob), tradução nossa.

Reply:
De fato! Indeed!